cc_licenses/settings/publish.py

```diff
@@ -0,0 +1,5 @@
+from settings import base
+
+from .base import *  # noqa
+
+DISTILL_DIR = f"{base.TRANSLATION_REPOSITORY_DIRECTORY}/build/"
```

Comment:
I don't think we need a separate settings file for publishing. The settings file we use is generally determined by where we are (local dev, staging, production) and not what we're doing.